analysers/trials/faban.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from datetime import timedelta
+
+import scipy.integrate as integrate
+import scipy.special as special
+import numpy as np
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+def createQuery(sc, cassandraKeyspace, srcTable, experimentID, trialID, containerID, hostID):
+    from commons import computeMode, computeMetrics
+    
+    queries = []
+    
+    dataRDD = sc.cassandraTable(cassandraKeyspace, srcTable) \
+            .select("value", "section", "host", "op_name") \
+            .where("trial_id=? AND experiment_id=?", trialID, experimentID) \
+            .cache() \
+            
+    hosts = dataRDD.map(lambda a: a["host"]).distinct().collect()
+    hosts.append("aggregate")
+    operations = dataRDD.map(lambda a: a["op_name"]).distinct().collect()
+    sections = ["WebDriver Throughput", "WebDriver Response Times"]
+    
+    for host in hosts:
```

Comment:
@VincenzoFerme I would need to go over the aggregate ones for all ops and sections if I do that. I can keep the looping like this, because all hosts have the same ops and sections, so I don't end up  wasting time in this case.